e2e/shortcut-customize-links.test.ts

```diff
@@ -0,0 +1,168 @@
+import { test, expect, type Page } from '@playwright/test';
+import { E2ESession } from './e2e-helpers';
+import Onboarding from './page-objects/onboarding';
+import SiteContent from './page-objects/site-content';
+import WhatsNewModal from './page-objects/whats-new-modal';
+import { getUrlWithAutoLogin } from './utils';
+
+const global = globalThis as unknown as {
+	__originalOpenExternal?: ( url: string ) => Promise< void >;
+	__openExternalCall?: string | null;
+};
+test.describe( 'Shortcut Customize links', () => {
+	const session = new E2ESession();
+
+	const siteName = 'E2E-Shortcuts-Site';
+
+	const mockOpenExternal = async () => {
+		await session.electronApp.evaluate( ( { shell } ) => {
+			if ( ! global.__originalOpenExternal ) {
+				global.__originalOpenExternal = shell.openExternal;
+			}
+			global.__openExternalCall = null;
+			shell.openExternal = async ( url: string ) => {
+				global.__openExternalCall = url;
+			};
+		} );
+	};
+
+	const getOpenExternalCall = async (): Promise< string | null > => {
+		return await session.electronApp.evaluate( () => {
+			return global.__openExternalCall ?? null;
+		} );
+	};
+
+	const restoreOpenExternal = async () => {
+		await session.electronApp.evaluate( ( { shell } ) => {
+			if ( global.__originalOpenExternal ) {
+				shell.openExternal = global.__originalOpenExternal;
+			}
+			delete global.__openExternalCall;
+			delete global.__originalOpenExternal;
+		} );
+	};
+
+	const openShortcut = async ( page: Page, label: string ) => {
+		const siteContent = new SiteContent( session.mainWindow, siteName );
+		await expect( siteContent.runningButton ).toBeAttached( { timeout: 120_000 } );
+
+		await mockOpenExternal();
+
+		await siteContent.locator.getByRole( 'button', { name: label } ).click();
+
+		const openedUrl = await getOpenExternalCall();
+
+		expect( openedUrl ).toBeTruthy();
+		if ( ! openedUrl ) {
+			throw new Error( 'openExternal was not called' );
+		}
+
+		await page.goto( getUrlWithAutoLogin( openedUrl ), {
+			waitUntil: 'domcontentloaded',
+		} );
+		return page.url();
+	};
+	describe( 'Block theme customize shortcuts', () => {
```

Comment:
I get this error when running the suite:

```
ReferenceError: describe is not defined

   at shortcut-customize-links.test.ts:65

  63 |          return page.url();
  64 |  };
> 65 |  describe( 'Block theme customize shortcuts', () => {
     |  ^
  66 |          test.beforeAll( async () => {
  67 |                  await session.launch();
  68 |
    at /Users/gcsecsey/a8c-projects/studio/e2e/shortcut-customize-links.test.ts:65:2
    at Object.<anonymous> (/Users/gcsecsey/a8c-projects/studio/e2e/shortcut-customize-links.test.ts:12:6)
Error: No tests found
```

When using `test.describe`, the suite passes:

<img width="1451" height="302" alt="Image" src="https://github.com/user-attachments/assets/fb2937a2-a78f-45be-bed7-e7f7a8a67bc8" />

```suggestion
	test.describe( 'Block theme customize shortcuts', () => {
```